Clarify documentation around lost Cypher updates (#119)
neo4j/docs-java-reference: modules/ROOT/pages/transaction-management.adoc

----
====

> Due to the complexity of determining such a dependency in the general case, Cypher does not cover all cases. If we execute one of the following queries conurrently 100 times, the final value of `n.prop` will most probably be less than 100.

Reviewer: ```suggestion
Due to the complexity of determining such a dependency in the general case, Cypher does not cover all cases.
If you run one of the following queries concurrently 100 times, the final value of `n.prop` will most probably be less than 100.
```
